src/test/java/com/box/sdk/BoxSignRequestTest.java

```diff
@@ -0,0 +1,268 @@
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.post(WireMock.urlPathEqualTo(requestUrl))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        BoxSignRequest signRequest = new BoxSignRequest(this.api, signRequestId);
+        BoxSignRequest.Info signRequestInfo = signRequest.cancel();
+
+        Assert.assertEquals(BoxSignRequest.BoxSignRequestStatus.Cancelled, signRequestInfo.getStatus());
+    }
+
+    @Test
+    @Category(UnitTest.class)
+    public void resendSignRequestSucceeds() {
+        final String signRequestId = "12345";
+
+        final String requestUrl = "/sign_requests/" + signRequestId + "/resend";
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.post(WireMock.urlPathEqualTo(requestUrl))
+                .willReturn(WireMock.aResponse()
+                        .withStatus(202)));
+
+        BoxSignRequest signRequest = new BoxSignRequest(this.api, signRequestId);
+
+        signRequest.resend();
+
+        WireMock.verify(1, postRequestedFor(urlPathEqualTo(requestUrl)));
+    }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void createSignRequestIntegrationTest() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        String signerEmail = "mwoda+staging@boxdemo.com";
+        List<BoxSignRequestSigner> signers = new ArrayList<BoxSignRequestSigner>();
+        BoxSignRequestSigner newSigner = new BoxSignRequestSigner(signerEmail)
+                .setInPerson(false);
+        signers.add(newSigner);
+
+        String fileId = "11438710730";
+        List<BoxSignRequestFile> files = new ArrayList<BoxSignRequestFile>();
+        BoxSignRequestFile file = new BoxSignRequestFile(fileId);
+        files.add(file);
+
+        String parentFolderId = "6732314306";
+
+        BoxSignRequestCreateParams createParams = new BoxSignRequestCreateParams()
+                .setIsDocumentPreparationNeeded(true);
+        BoxSignRequest.Info signRequestInfo = BoxSignRequest.createSignRequest(api, signers,
+                files, parentFolderId, createParams);
+
+        BoxFile.Info fileInfo = signRequestInfo.getSourceFiles().get(0);
+        BoxSignRequestSigner signer = signRequestInfo.getSigners().get(0);
+
+        Assert.assertNotNull(signRequestInfo.getPrepareUrl());
+        Assert.assertEquals(fileId, fileInfo.getID());
+        Assert.assertEquals(signerEmail, signer.getEmail());
+        Assert.assertNotNull(signRequestInfo.getID());
+    }
+
+    @Test
+    @Category(IntegrationTest.class)
+    public void getSignRequestIntegrationTest() {
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        final String fileId = "11438710730";
+        final String signerEmail = "mwoda+staging@boxdemo.com";
+        final String signerName = "Mateusz Woda";
```

Comment:
Don't merge this information. But, since it's already been compromised (since it's part of this commit), you should change the email address on this account.